.github/workflows/progress-tracker.yml

```
@@ -205,8 +205,13 @@ jobs:
               ':-----------------|------:|----:|----:|-----:|------:'
             ].concat(Object.entries(teamStats).map(([k, v]) => `${k} | ${v.join(' | ')}`)).join('\n')
 
-            const teams = Object.entries(teamMembers).map(([k, v]) => {
-              return `### ${k} (${Object.keys(v).length})\n\n${Object.entries(v).sort().map(([m, c]) => `<span title="Chapters: ${c.join(', ')}">${m} (${c.length})</span>`).join(', ')}`
+            const teams = Object.entries(teamMembers).map(([team, contributors]) => {
```

This will ensure usernames are sorted withing the group of same number of contributions.

```suggestion
            const teams = Object.entries(teamMembers).sort().map(([team, contributors]) => {
```

---

👍

---

I think this sorts teams, not members. I moved this down before the sort by length and made it a case insensitive sort for convenience.

Also, I realized that I can test this action in this repo before merging by setting the branch in the "run workflow" manual run. You can see the results in #989.

---

> Also, I realized that I can test this action in this repo before merging by setting the branch in the "run workflow" manual run.

This is a very good catch, I totally forgot about this option.